addons/sale/models/sale.py

```diff
@@ -676,7 +676,7 @@ def _get_invoiceable_lines(self, final=False):
 
         return self.env['sale.order.line'].browse(invoiceable_line_ids + down_payment_line_ids)
 
-    def _create_invoices(self, grouped=False, final=False, date=None):
+    def _create_invoices(self, grouped=False, final=False, **kwargs):
```

Comment:
Although this is a good generic approach, this should be put since the beginning, as now dependent modules still depends on the old signature, so with this, you are forcing to change the signature of these modules to add the `**kwargs` part if we don't want a crash. Example:

https://github.com/OCA/account-invoicing/blob/14.0/sale_order_invoicing_grouping_criteria/models/sale_order.py#L37

Reply:
@pedrobaeza Thanks for the feedback, you're right.  :)

Reply:
@pedrobaeza I pushed a new version that should be ok with another override of the form
```py
def _create_invoices(self, grouped=False, final=False, date=None):
    # DO STUFF
    return super()._create_invoice(grouped=grouped, final=final, date=date)
```
Is it ok for you ?

Reply:
Well, this will still miss extra kwargs when installing `sale_timesheet` module provoking incorrect behavior, but if you don't like the context solution, go ahead with this and at least I'll add `**kwargs` on my known modules.